sklearn/mixture/tests/test_gmm.py

```diff
@@ -104,6 +106,17 @@ def test_lmvnpdf_full():
     assert_array_almost_equal(lpr, reference)
 
 
+def test_lvmpdf_full_cv_non_positive_definite():
+    n_features, n_components, n_samples = 2, 1, 10
+    X = rng.randint(10) * rng.rand(n_samples, n_features)
```

Comment:
I would prefer if you could use a random number generator local to the test: on the line above, insert
<pre>
 rng = np.random.RandomState(0)
</pre>